<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RichTextEdit</class>
 <widget class="QWidget" name="RichTextEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="f_toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="RSComboBox" name="f_paragraph">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Paragraph formatting</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="RSComboBox" name="f_fontsize">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Font size</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_fgcolor">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Text foreground color</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_bgcolor">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Text background color</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_undo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Undo (CTRL+Z)</string>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/undo.png</normaloff>:/icons/textedit/undo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_redo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Redo</string>
        </property>
        <property name="text">
         <string>Redo</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/redo.png</normaloff>:/icons/textedit/redo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_cut">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Cut (CTRL+X)</string>
        </property>
        <property name="text">
         <string>Cut</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/cut.png</normaloff>:/icons/textedit/cut.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_copy">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Copy (CTRL+C)</string>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/copy.png</normaloff>:/icons/textedit/copy.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_paste">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Paste (CTRL+V)</string>
        </property>
        <property name="text">
         <string>Paste</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/paste.png</normaloff>:/icons/textedit/paste.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_link">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Link (CTRL+L)</string>
        </property>
        <property name="text">
         <string>Link</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/link.png</normaloff>:/icons/textedit/link.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_bold">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string notr="true">Bold (CTRL+B)</string>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/bold.png</normaloff>:/icons/textedit/bold.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_italic">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Italic (CTRL+I)</string>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/italic.png</normaloff>:/icons/textedit/italic.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_underline">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Underline (CTRL+U)</string>
        </property>
        <property name="text">
         <string>Underline</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/underline.png</normaloff>:/icons/textedit/underline.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_strikeout">
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/strikethrough.png</normaloff>:/icons/textedit/strikethrough.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_list_bullet">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Bullet list (CTRL+-)</string>
        </property>
        <property name="text">
         <string>Bullet list</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/bullet-list.png</normaloff>:/icons/textedit/bullet-list.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_list_ordered">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Ordered list (CTRL+=)</string>
        </property>
        <property name="text">
         <string>Ordered list</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/numberd-list.png</normaloff>:/icons/textedit/numberd-list.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_indent_dec">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Decrease indentation (CTRL+,)</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/left-indent.png</normaloff>:/icons/textedit/left-indent.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_indent_inc">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Increase indentation (CTRL+.)</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/right-indent.png</normaloff>:/icons/textedit/right-indent.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_image">
        <property name="toolTip">
         <string>Attach a Picture</string>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/photo-of-a-landscape.png</normaloff>:/icons/textedit/photo-of-a-landscape.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="f_menu">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../gui/icons.qrc">
          <normaloff>:/icons/textedit/settings.png</normaloff>:/icons/textedit/settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>f_paragraph</zorder>
     <zorder>f_undo</zorder>
     <zorder>f_redo</zorder>
     <zorder>f_cut</zorder>
     <zorder>f_copy</zorder>
     <zorder>f_paste</zorder>
     <zorder>line</zorder>
     <zorder>f_link</zorder>
     <zorder>line_3</zorder>
     <zorder>f_italic</zorder>
     <zorder>f_underline</zorder>
     <zorder>line_2</zorder>
     <zorder>line_5</zorder>
     <zorder>f_list_bullet</zorder>
     <zorder>f_list_ordered</zorder>
     <zorder>f_indent_dec</zorder>
     <zorder>f_indent_inc</zorder>
     <zorder>f_bold</zorder>
     <zorder>f_strikeout</zorder>
     <zorder>f_image</zorder>
     <zorder>f_menu</zorder>
     <zorder>f_fontsize</zorder>
     <zorder>f_fgcolor</zorder>
     <zorder>f_bgcolor</zorder>
     <zorder>line_4</zorder>
    </widget>
   </item>
   <item>
    <widget class="MimeTextEdit" name="f_textedit">
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
      </font>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="f_info">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RSComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/common/RSComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>MimeTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">gui/common/MimeTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>f_textedit</tabstop>
  <tabstop>f_strikeout</tabstop>
  <tabstop>f_image</tabstop>
  <tabstop>f_menu</tabstop>
 </tabstops>
 <resources>
  <include location="../gui/icons.qrc"/>
 </resources>
 <connections/>
</ui>
